stdlib/grpc/src/main/java/org/ballerinalang/net/grpc/proto/ServiceProtoBuilder.java

```diff
@@ -167,9 +181,13 @@ private void addDescriptorAnnotation(ServiceNode serviceNode, File fileDefinitio
 
         BLangLiteral valueLiteral = null;
         LiteralNode literalNode1 = TreeBuilder.createLiteralExpression();
-        if (literalNode1 instanceof BLangLiteral && fileDefinition != null) {
+        if (literalNode1 instanceof BLangLiteral) {
```

Comment:
Shall we avoid using `instanceof` and check the node kind instead?
i.e. `literalNode1.getKind() == NodeKind.LITERAL`
Also maybe rename literalNode1